### DISCUSSION TITLE
Block the update of WebUI JupyterLab

### DISCUSSION THREAD

**Discussion by agrosu1978** [jupyterlab/jupyterlab#15114] [Q&A]
Hello

We have a docker image which includes the JupyterLab WebUI.  It is started as a container from a jupyterhub link. Some users have experienced problems starting their Jupyter server. The problem seems linked to an update of the JupyterLab (WebUI) version. Thus, when the docker container is created, the server does not start. I am wondering how to block the possibility of updating the version of JupyterLab. I searched a lot here on github project discussions, but I did not find anything to guide me through this
Thank you a lot for any help!

**Comment by krassowski** [comment]
JupyterLab does not auto-update itself. Your issue must be with the package manager which installs it.

**Reply by agrosu1978** [reply]
I haven't said that JupyterLab is auto-updating, sorry for not making myself clearer. The WebUI is offering the possibility at some point to update, and the user is doing it. I think it is related to the "Check for JupyterLab updates" from Notification (please have a look at the attached pic). So I am wondering if by configuration I may set this parameter to False, thus the user will not be asked to perform an update anymore. This will be done manually only.
![jupytelab_notification](https://github.com/jupyterlab/jupyterlab/assets/69220930/7bd811b2-56b8-441d-9cdb-7cba14fbc0a3)

**Reply by krassowski** [reply]
Please read the documentation: https://jupyterlab.readthedocs.io/en/latest/user/announcements.html#check-for-updates
If anything is not clear please open an issue and contribute back to the documentation :)
